MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -1644,13 +2213,13 @@ public static Person generateInstanceFromXML(Node wn, Campaign c, Version versio
                     String systemId = wn2.getAttributes().getNamedItem("systemId").getTextContent().trim();
                     String planetId = wn2.getTextContent().trim();
                     retVal.originPlanet = c.getSystemById(systemId).getPlanetById(planetId);
-                } else if (wn2.getNodeName().equalsIgnoreCase("isClanTech")
-                           || wn2.getNodeName().equalsIgnoreCase("clan")) {
+                } else if (wn2.getNodeName().equalsIgnoreCase("clan")) {
                     retVal.clan = Boolean.parseBoolean(wn2.getTextContent().trim());
                 } else if (wn2.getNodeName().equalsIgnoreCase("phenotype")) {
                     retVal.phenotype = Integer.parseInt(wn2.getTextContent());
                 } else if (wn2.getNodeName().equalsIgnoreCase("bloodname")) {
                     retVal.bloodname = wn2.getTextContent();
+                    retVal.setFullName();
```

Comment:
Why not just call this once at the very end?

Reply:
Switched it over to do so